.travis.yml

```diff
@@ -18,3 +18,14 @@ notifications:
 #  - julia -e 'using Pkg; Pkg.add(pwd()); Pkg.build("Clustering"); Pkg.test("Clustering"; coverage=true)'
 after_success:
   - julia -e 'using Pkg, Clustering; cd(joinpath(dirname(pathof(Clustering)), "..")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
+
+jobs:
+  include:
+    - stage: "Documentation"
+      julia: 1.1
+      os: linux
+      script:
+        - julia --project=doc/ -e 'using Pkg; Pkg.instantiate();
+                                    Pkg.develop(PackageSpec(path=pwd()))'
```

Comment:
If I'm not mistaken, it's better to `develop` before `instantiating`, otherwise you can get weird version resolution issues in some cases.